cogs/recruit.py

```diff
@@ -143,6 +175,35 @@ async def purge(self, ctx: commands.Context):
 
         await ctx.reply("Done.")
 
+    @commands.hybrid_command(name="exceptions", with_app_command=True, description="Add or remove exceptions")
+    @app_commands.guilds(configInstance.data.guild)
+    @commands.has_permissions(administrator=True)
```

Comment:
Iiiick.  When are we going to get rid of these things that require Administrator?

Reply:
I'll add that to the to do list